bin/build

```diff
@@ -10,16 +10,24 @@ MIQ_ORG=${MIQ_ORG:-ManageIQ}
 SUI_ORG=${SUI_ORG:-ManageIQ}
 APPLIANCE_ORG=${APPLIANCE_ORG:-ManageIQ}
 
+REPO_PREFIX=${REPO_PREFIX:-manageiq}
+
+GIT_HOST=${GIT_HOST:-github.com}
+GIT_TOKEN=${GIT_TOKEN:-""}
+
 NPM_REGISTRY_OVERRIDE=${NPM_REGISTRY_OVERRIDE:-""}
 
-while getopts "t:d:r:nhp" opt; do
+while getopts "t:d:f:g:k:r:nhp" opt; do
   case $opt in
     d) IMAGE_DIR=$OPTARG ;;
+    f) REPO_PREFIX=$OPTARG ;;
+    g) GIT_HOST=$OPTARG ;;
+    k) GIT_TOKEN=$OPTARG ;;
```

Comment:
I'm struggling to come up with meaningful option letters for these ... maybe they should all be driven by environment variables like `NPM_REGISTRY_OVERRIDE`.